internal/install/psutil_discoverer.go

```diff
@@ -53,10 +53,11 @@ func (p *psUtilDiscoverer) discover(ctx context.Context) (*discoveryManifest, er
 		processes = append(processes, psUtilProcess(*pp))
 	}
 
-	filtered, err := p.processFilterer.filter(processes)
-	if err != nil {
-		return nil, err
-	}
+	filtered := processes
```

Comment:
No filtering for now?

Reply:
The recipe service hasn't exposed the filters yet to be able to drive this.